test/e2e/main_test.go

```diff
@@ -78,6 +78,21 @@ func TestBrokerDirect(t *testing.T) {
 	env.Finish()
 }
 
+// TestVhostBrokerDirect makes sure a Broker can delivery events to a consumer inside a rabbitmq vhost.
+func TestVhostBrokerDirect(t *testing.T) {
+	t.Parallel()
+	ctx, env := global.Environment(
+		knative.WithKnativeNamespace(system.Namespace()),
+		knative.WithLoggingConfig,
+		knative.WithTracingConfig,
+		k8s.WithEventListener,
+	)
+	env.Test(ctx, t, RabbitMQClusterVHost())
```

Comment:
I can't find the feature that adds a Rabbitmq cluster with vhost. Are there files that were missed with this PR?

Reply:
Im reusing the old feature that I used with the source to create a rabbitmq cluster with a default vhost